### ISSUE TITLE
Screen reader enabled during install but not enabled after install

### ISSUE THREAD

**Issue by jbicha** [canonical/ubuntu-desktop-installer#2343]
### What happened?

Use the new Ubuntu Desktop Installer and enable the screen reader to complete the install. After the install is complete, the screen reader is not activated for the new install

### What was expected?

With the Legacy Installer, installing with the screen reader turned on enables the screen reader for both the login screen and for (at least) the first user.

### Steps to reproduce

- Boot the installer ISO
- After it plays the welcome sound, press Super+Alt+S to turn on the screen reader
- Complete the installation with the screen reader turned on
- When the install is complete, restart the computer
- The login screen should have the screen reader enabled

### Additional context

20230921.1 Ubuntu Desktop Mantic ISO

**Comment by seb128** [comment]
Updating the title to match the description, the configuration is correctly carried to the installed system user session but the issue is about the login screen
